src/services/fileServices/YmlFileServices/ConfigYmlService.js

```diff
@@ -17,7 +17,13 @@ class ConfigYmlService {
         fileName: CONFIG_FILE_NAME,
       }
     )
-    const content = sanitizedYamlParse(unparsedContent)
+    const sanitisedContent = sanitizedYamlParse(unparsedContent)
+    const content = {
+      ...sanitisedContent,
+      "facebook-pixel": sanitisedContent["facebook-pixel"]
+        ? parseInt(sanitisedContent["facebook-pixel"], 10)
+        : null,
```

Comment:
note that `parseInt` can give `NaN`, which has to be handled appropriately (or are we letting the frontend do that?)

Reply:
Curious, what are your thoughts on disruptively modifying their repo if it cannot be parsed? That way we can have more assurance in the future that is has to be either an int or a string that can be parsed into an int

Reply:
Handled in https://github.com/isomerpages/isomercms-backend/pull/776/commits/7fec5ed2d56cb54455a094b8f1dec4f2d01c4557

I think we should be disruptively modifying their repo - this change causes any future change to settings to also modify the type of facebook-pixel where possible. However, we still won't be able to guarantee that this is always true - it's possible for sites to never touch settings and thus not make this change.